src/utils/capmodpp.hpp

```diff
@@ -0,0 +1,151 @@
+#pragma once
```

Comment:
:wrench: 
ok, well the file including this file is supposed to test.
added #error